zzz_generated/app-common/app-common/zap-generated/cluster-objects.cpp

```diff
@@ -0,0 +1,411 @@
+#include "cluster-objects.h"
+
+#pragma GCC diagnostic ignored "-Wstack-usage="
+
+namespace chip {
+namespace app {
+namespace clusters {
+namespace TestCluster {
+namespace TestListStructOctet {
+CHIP_ERROR Type::Encode(TLV::TLVWriter & writer, uint64_t tag)
+{
+    TLV::TLVType outer;
+    ReturnErrorOnFailure(writer.StartContainer(tag, TLV::kTLVType_Structure, outer));
+    ReturnErrorOnFailure(DataModel::Encode(writer, TLV::ContextTag(kFabricIndexFieldId), fabricIndex));
```

Comment:
@mrjerryjohns maybe we can consider to use ReturnLogErrorOnFailure? to log error in the initial place?

Reply:
That's quite expensive, and will be multiplied across every single call in every single cluster object.

If we find a need for it later, let's think about adding it then.